protocol/app/ante/timestampnonce.go

```diff
@@ -0,0 +1,55 @@
+package ante
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	ante "github.com/cosmos/cosmos-sdk/x/auth/ante"
+	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+	accountpluskeeper "github.com/dydxprotocol/v4-chain/protocol/x/accountplus/keeper"
+)
+
+// TODO: combine increment sequence and sequence verification into one decorator
+// https://github.com/cosmos/cosmos-sdk/pull/18817
+type IncrementSequenceDecorator struct {
+	ak ante.AccountKeeper
+}
+
+func NewIncrementSequenceDecorator(ak ante.AccountKeeper) IncrementSequenceDecorator {
```

Comment:
What's the rationale behind forking this decorator here, vs updating it in the `dydxprotocol/cosmo-sdk` fork?

Reply:
I was under the impression that updating the fork required additional release complexity. 

There is actually a ticket to deprecate IncrementSequenceDecorator altogether by adding the incrementation into sigverify. (https://linear.app/dydx/issue/OTE-624/combine-incrementsequencedecorator-into-sigverificationdecorator)

I think I will have time to address this ticket and the other comments before v6 release. Will publish new PR soon